src/main/java/org/sagebionetworks/web/client/widget/team/JoinTeamWidget.java

```diff
@@ -236,16 +284,16 @@ public int getTotalPageCount() {
 	 * Gather additional info about the logged in user
 	 */
 	public void sendJoinRequestStep1(WikiPageKey challengeInfoWikiPageKey) {
-		UserSessionData sessionData = authenticationController.getCurrentUserSessionData();
-		UserProfile profile = sessionData.getProfile();
-		view.updateWizardProgress(currentPage, getTotalPageCount());
-		view.showChallengeInfoPage(profile, challengeInfoWikiPageKey, new Callback() {
+		progressWidget.configure(currentPage, getTotalPageCount());
```

Comment:
Looks good to me.
![image](https://cloud.githubusercontent.com/assets/8346437/9452787/19f008b0-4a6b-11e5-82ea-d49cd58e9588.png)